ui/packages/components/src/Info/Info.tsx

```diff
@@ -11,20 +11,24 @@ export const Info = ({
   action,
   iconElement,
   widthClassName = 'max-w-xs',
+  side,
```

Comment:
We've been usually assigning defaults in destructuring. It might be easier to read `side='right' `here in the function signature instead of `side={side ?? 'right'}` inside the PopoverContent